docker/scripts/deploy-contracts.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -eux
+
+echo "JSONRPC_HTTP_URL: $JSONRPC_HTTP_URL"
+node docker/scripts/fund-accounts.js
+echo "Replace 1001 with $L2_CHAIN_ID"
+sed -i.bak "s/\"chainID\": 1001/\"chainID\": $L2_CHAIN_ID/g" docker/scripts/deploy_parameters_docker.json
```

Comment:
let's first update `deploy_parameters_docker.json` to contain:

```
"chainID": <chainID>,
```

so here you just need to replace it to `$L2_CHAIN_ID`